test-requirements.txt

```diff
@@ -10,3 +10,4 @@ testtools>=2.3.0 # MIT
 tomli>=1.1.0;python_version<"3.11" # MIT
 beautifulsoup4>=4.8.0 # MIT
 pylint==1.9.4 # GPLv2
+GitPython # BSD License (3 clause)
```

Comment:
I don't know why we have tomli and GitPython here but not pyyaml. (I'm guessing PyYAML is transitively installed elsewhere.)

I would rather replace those with:

```
-e .[yaml,toml,baseline]
```

As that will install whatever the dependencies are for the extras in our metadata, not just what we remembered to add here.